docs/build-acceleration.md

```diff
@@ -61,6 +63,73 @@ Setting _Logging Level_ to a value other than `None` results in messages prefixe
 - `Minimal` produces a single message per out-of-date project.
 - `Info` and `Verbose` provide increasingly detailed information about the inner workings of the check, which are useful for debugging.
 
+### Validate builds are accelerated
+
+If build acceleration cannot be enabled for any of the reasons given below, builds continue to work as before but will be slower.
+
+The following prerequisites exist for build acceleration:
+
+- You are running Visual Studio 2022 version 17.5 or later.
+- The project is an SDK-style .NET project.
```

Comment:
Have we tested this for VB/F#?

Reply:
VB: yes, because we are dogfooding it in this repo.
F#: not yet. I'll make sure that happens.